index.html

```diff
@@ -1,26 +1,22 @@
 <html lang="en">

```

Comment:
![image](https://user-images.githubusercontent.com/44003977/218644751-2300bc06-cb7a-44f7-a166-701ad4ba21f2.png)

The line is crossing the `y` letter in a word, but shouldn't be crossing the word according to the design